test/mongoDbTest.js

```diff
@@ -0,0 +1,43 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoDb = require('../mongoDb/index.js');
+
+const should = chai.should();
+
+chai.use(chaiHttp);
+
+const id = Math.floor(Math.random() * 300000);
+
+describe('UserMovies', () => {
+
+  it('should contain at least 1,000,000 users', () => {
+    mongoDb.UserMovies.count({}, (err, result) => {
+    })
+      .then((count) => {
+        const isBigEnough = count >= 1000000;
```

Comment:
I like this test! Doesn't require dropping the table every time 👍

Reply:
thanks